winter/http/response_header_resolver.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.request import Request
+
+from winter.http.response_header import _BaseResponseHeader
+from .response_header import ResponseHeaderAnnotation
+from .response_header import ResponseHeaders
+from .. import ArgumentResolver
+from ..core import ComponentMethodArgument
+
+
+class ResponseHeaderArgumentResolver(ArgumentResolver):
+
+    def is_supported(self, argument: ComponentMethodArgument) -> bool:
+        if not isinstance(argument.type_, type) or not issubclass(argument.type_, _BaseResponseHeader):
```

Comment:
the same as above: isinstance(typing.Dict, type) is False

Reply:
fixed